client/cpp/kserve-api/samples/grpc_async_infer_resnet.cpp

```diff
@@ -69,7 +69,12 @@ std::vector<uint8_t> load(const std::string& fileName) {
     fileImg.seekg(0, std::ios::beg);
 
     char* buffer = new char[bufferLength];
-    fileImg.read(buffer, bufferLength);
+    try {
+        fileImg.read(buffer, bufferLength);
+    }
+    catch(std::bad_alloc) {
```

Comment:
make style should validate missing space after catch
why do we catch just to throw anyway?